### DISCUSSION TITLE
Represent Relation as a different field

### DISCUSSION THREAD

**Discussion by normanmadrid** [pocketbase/pocketbase#1554] [Q&A] [answered]
Hi! Love pocketbase btw! so many great features out of the box, however I can't seem to find if its at all possible to configure how a relation is represented inside the collections table in dashboard, I'm trying to get a relation to be represented as another field other than ID in the collection its related to:
![image](https://user-images.githubusercontent.com/71151590/211403808-a27d96af-f5aa-47d3-9afa-289db5b72c65.png)

ex. my accounts collection table, i want to see the relation contacts as the email field of contacts and not the ID of the contact would very much appreciate the helping hand! thanks!

**Comment by ganigeorgiev** [comment, marked as answer]
At the moment this is not possible but will be available with the next v0.12.0 release as part of https://github.com/pocketbase/pocketbase/issues/976.

**Reply by normanmadrid** [reply]
wow you DO reply fast haha, thank you so much for replying with haste, look forward to v0.12.0!
